core/trino-main/src/main/java/io/trino/operator/OperationTimer.java

```diff
@@ -60,12 +60,6 @@ class OperationTimer
         intervalCpuStart = cpuStart;
     }
 
-    void resetInterval()
```

Comment:
Without any form of reset, this feels wrong:

https://github.com/trinodb/trino/blob/259e2af70f4d1475cf7205dcc12bb45aab2b047c/core/trino-main/src/main/java/io/trino/operator/Driver.java#L403-L404

it looks like we're accounting all time since previous addInput as a time spent in the next addInput, i..e measuring wall time till last addInput. 

did something change recently? or am i just reading this wrong?

Reply:
that methods was used only by work processor adapter